Loop/Managers/DeviceDataManager.swift

```diff
@@ -1341,8 +1353,15 @@ extension Notification.Name {
 
 // MARK: - Remote Notification Handling
 extension DeviceDataManager {
```

Comment:
High-level changes in this class:

1. Make an async variant of `handleRemoteNotification`. This will be especially helpful for remote commands 2 later.
2. Use the new RemoteAction models for enacting a command. 
3. Add methods for handling async background uploads